docker/configure_workers_and_start.py

```diff
@@ -476,6 +478,39 @@ def generate_worker_files(
     # which exists even if no workers do.
     healthcheck_urls = ["http://localhost:8080/health"]
 
+    # Expand worker_type multiples if requested in shorthand(e.g. worker:2). Checking
+    # for not an actual defined type of worker is done later.
+    # Checking performed:
+    # 1. if worker:2 or more is declared, it will create additional workers up to number
+    # 2. if worker:1, it will create a single copy of this worker as if no number was
+    #   given
+    # 3. if worker:0 is declared, this worker will be ignored. This is to allow for
+    #   scripting and automated expansion and is intended behaviour.
+    # 4. if worker:NaN or is a negative number, it will error and log it.
+    new_worker_types = []
+    for worker_type in worker_types:
+        if ":" in worker_type:
+            x = worker_type.split(":")
+            y = 0
+            # should only be 2 components, a type of worker and an integer as a
+            # string. Cast the number as an int then it can be used as a counter.
+            if len(x) == 2 and x[-1].isdigit():
+                y = int(x[1])
+            else:
+                error(
+                    "Multiplier signal(:) for worker found, but incorrect components: "
+                    + worker_type
+                )
+            # As long as there are more than 0, we add one to the list to make below.
+            while y > 0:
```

Comment:
`for _ in range(worker_count):` would be fine for this

Reply:
Neat. Didn't know I could do that. Fixed
(Also the `x, y` did get renamed)